tests/4.5/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_map_tofrom.F90

```diff
@@ -0,0 +1,63 @@
+!--- test_target_teams_distribute_parallel_for_map_tofrom.F90 ---------------===//
+! 
+! OpenMP API Version 4.5 Nov 2015
+!
+! Testing the mapping of arrays and scalars through the map clause with
+! the 'tofrom' map-type.
+! 
+!//===-----------------------------------------------------------------------===//
+#include "ompvv.F90"
+
+#define N 1000
+
+PROGRAM test_target_teams_distribute_parallel_for_map_tofrom
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+  OMPVV_TEST_VERBOSE(target_teams_distribute_parallel_for_map_tofrom() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+   INTEGER FUNCTION target_teams_distribute_parallel_for_map_tofrom()
+      INTEGER :: scalar_to, scalar_from, errors, i, j 
+      INTEGER, DIMENSION(N) :: a, b, c, d
+      OMPVV_INFOMSG("test_target_teams_distribute_parallel_for_map_tofrom");
+      
+      scalar_to = 50
+      scalar_from = 50
+      errors = 0
+      
+      DO i = 1, N
+         a(i) = 1
+         b(i) = i
+         c(i) = 2*i
+         d(i) = 0
+      END DO
+
+      !$omp target teams distribute parallel do map(tofrom: a, b, c, d,&
+      !$omp& scalar_to, scalar_from)
```

Comment:
a bit confused on the formatting on this one, what purpose does line 42 have?

Reply:
The & is a way to split the pragma across multiple lines for cleanliness sake. The compiler sees this is one single pragma `!$omp target teams distribute parallel do map(tofrom: a, b, c, d, scalar_to, scalar_from)`